staging/src/k8s.io/component-base/zpages/flagz/flagz.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 const (
-	flagzHeaderFmt = `
+	FlagzHeaderFmt = `
```

Comment:
we should really pre-generate some complete expected outputs and test against that instead of re-using the implementation details in the test?

Reply:
I thought reusing this would be a good idea since it is the "expected" outputs, and if there is a change here we don't need to bother to change the expected raw output somewhere else, no?

Reply:
We aren't actually testing the output if we just run the format in two places?

We also probably don't want to make the header format string a public API anyhow

Reply:
> We aren't actually testing the output if we just run the format in two places?

Hmm rethinking this now,  this actually makes sense. This seems like a target case for https://testing.googleblog.com/2014/07/testing-on-toilet-dont-put-logic-in.html

Reply:
SG, changed it back. PTAL again.